ASM/README.md

```diff
@@ -58,14 +58,16 @@ You'll need to disable `Unique Game Save Directory` in Project64 for these to wo
 
 How to use the Debug mode: 
 - First put the DEBUG_MODE variable at 1 in debug.h.
-- You will then have access to a hidden menu with the following options:
+- Now the N64 logo sequence at the start of the game will be skipped, the L button will allow you to levitaten and you will then have access to a hidden menu with the following options:
   - Instant warps to Dungeons, Bosses or Overworld locations
   - Item inventory edits
   - Instant age switch with the current location kept
-  - Bunny Hood applied on
-The menu will appear if you press either L+R or Dpad-Up according to what you set for the menu_not_on_dup variable in debug.c.
+  - Bunny Hood
+  - In-game clock
+  - Actor and overlay list
+  - Scene flags setters
```

Comment:
I'm unclear what this does, when I select any of the options nothing happens visibly. I went into a room with chests and selected the "Treasure" option after opening one of them, still to no effect

Reply:
I'm not exactly a scene flag expert so take that with a grain of salt, but i think the treasure flags are only used to mark chests as already opened.
They don't have an Unset so once they're open, you can't change them back.
See in the video, setting the flag to 1 marks the chest as opened (and obviously you can't get the item anymore)

https://github.com/user-attachments/assets/6ddf428c-5b65-4a6f-b89d-88e075de3852

Reply:
Gotcha, that makes sense